src/Table/Table.test.tsx

```diff
@@ -262,7 +262,7 @@ describe("Component: Table", () => {
         expect(onRowSelected).toHaveBeenCalledTimes(4);
     });
 
-    describe("should handle inline textbox edit ", async () => {
+    describe("should handle inline textbox edit ", () => {
```

Comment:
😏

Reply:
`async` is not supported for `describe` statements

Reply:
😂